easybuild/easyconfigs/m/MATLAB/MATLAB-2022a_Update_3.eb

```
@@ -0,0 +1,26 @@
+name = 'MATLAB'
+version = '2022a_Update_3'
+
+homepage = 'https://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = SYSTEM
+
+sources = ['R%(version)s_Linux.iso']
```

Then the sources can be written as
```suggestion
sources = ['R%s_Update_%s_Linux.iso' % (version, _update)]
```